content/blog/2020-05-26-may-20-community-gems.md

```diff
@@ -0,0 +1,93 @@
+  and using DVC with teams.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  development best practices, sharing models and data across projects, 
+  and using DVC with teams.
+picture: 2020-05-26/May_20_Gems_Header.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/april-20-community-gems/356
+tags:
+  - Discord
+  - Gems
+  - Cache
+  - Google Cloud Storage
+  - Import
+---
+
+## Discord gems
+
+Here are some Q&A's from our Discord channel that we think are worth sharing.
+
+### Q: [How do I completely delete a file from DVC?](https://discord.com/channels/485586884165107732/563406153334128681/710546561498873886)
+
+You can simply delete a `.dvc` file to remove its target from DVC tracking, and
+then run `dvc gc` to
+[clear your DVC cache](https://dvc.org/doc/command-reference/gc#gc). If you want
+to remove a single `.dvc` file without doing a cache cleanup, look into the
+`.dvc` file and note the `md5` field inside. Then use this value to identify the
+corresponding file in your DVC cache and delete it. For example: if your target
+file has `md5: 123456`, the corresponding file in your cache will be
+`.dvc/cache/12/3456`.
+
+In the future, we plan to support a function like `git rm` that will allow for
+more granular delete options.
+
+### Q: [I'm using an S3 bucket as a DVC remote; the bucket is shared with several coworkers, who have projects in other folders in the bucket. Will it disrupt DVC if I manually add a README inside the folder being used for remote storage, to let my coworkers know what the directory is used for?](https://discord.com/channels/485586884165107732/563406153334128681/707551737745244230https://discord.com/channels/485586884165107732/563406153334128681/707551737745244230)
+
+That will be fine! Having an additional file in the remote that isn't part of
+DVC tracking won't pose any issues. You would only encounter problems if you
+were manually modifying or deleting contents of the remote managed by DVC.
+
+### Q: [Are there limits to how many files DVC can handle? My dataset contains ~100,000 files.](https://discord.com/channels/485586884165107732/563406153334128681/706538115048669274)
+
+We ourselves have stored datasets containing up to 2 million files, so 100,000
+is certainly feasible. Of course, the larger your dataset, the more time data
+transfer operations will take. Luckily, we have a
+[DVC 1.0 contains several data transfer optimizations](https://dvc.org/blog/dvc-3-years-and-1-0-release#data-transfer-optimizations)
+to substantially reduce the time needed to `dvc pull / push / status -c / gc -c`
+for very large datasets.
+
+### Q: [Two developers on my team are doing `dvc push` to the same remote. Should they `dvc pull` first?](https://discord.com/channels/485586884165107732/563406153334128681/704211629075857468)
+
+We encourage users to think of a DVC project like using Git- the same workflow
```

Comment:
I think we can keep the simplest version here- it's safe to push simultaneously, any files, no pull is needed. Also, in this particular case I would be very careful comparing DVC to Git. When it comes to data management DVC broke quite a few conventions comparing to Git - e.g.  it does not pull the whole history by default, it provides quite explicit interface to pull a specific file/directory. All of this because pulling can be quite expensive.

Reply:
Simplified to: 
```
It's safe to push simultaneously, no `dvc pull` needed. While some teams might be in the habit of frequently pulling, like in Git flow, there are less risks of "merge conflicts" in DVC.  That's because DVC remotes store files indexed by `md5`s, so there's
a very low probability of a collision (if two developers have two
different versions of a file, they'll be stored as two separate files in the DVC
remote- so no merge conflicts). 
```

if you still think it's too much, I can simplify to just the first sentence or so.